conflation/__version__.py

```diff
@@ -0,0 +1,6 @@
+__version__ = "0.0.1"
+__description__ = "Approximate driving speeds on location-specific road classes using public GPS trace data."
+__author__ = "Richard Chen"
+__author_email__ = ""
+__url__ = "https://github.com/OpenStreetMapSpeeds/conflation"
+__title__ = "conflation"
```

Comment:
best filled out by you guys

Reply:
Would you like me to do a PR into this branch to fill this out? Or I can fill this out in my next PR

Reply:
totally ok to do it iteratively and get it in the next round, no rush to have everything perfect in the first go. im just glad to see so much momentum here!